src/cms/views/events/event_view.py

```diff
@@ -172,7 +172,9 @@ def post(self, request, **kwargs):
             # Add error messages
             event_form.add_error_messages(request)
             event_translation_form.add_error_messages(request)
-            recurrence_rule_form.add_error_messages(request)
+            # do not call recurrence rule form clean method when recurrence rule is not set
+            if event_form.cleaned_data["is_recurring"]:
+                recurrence_rule_form.add_error_messages(request)
```

Comment:
Ok, so this one way to make sure that in case of an invalid event form the recurrence rule form is not validated, when the rule is not set. This is because `add_error_messages` will trigger a call to the recurrence rule forms clean method, which is not expected when the recurrence rule is disabled